src/main/java/org/fogbowcloud/saps/engine/core/archiver/storage/PermanentStorage.java

```diff
@@ -25,4 +26,6 @@
 	 */
 	boolean delete(SapsImage task) throws PermanentStorageException;
 
+	List<AccessLink> generateAccessLinks(String taskId) throws PermanentStorageException;
```

Comment:
doc-it. Is this list a collection of the link for each output file generated by a task? what-if this taskId was not found (or has no output generated)?
when does this exception is thown?

Reply:
Done.